storm-core/src/jvm/org/apache/storm/utils/Time.java

```diff
@@ -98,10 +104,25 @@ public static long currentTimeMillis() {
             return System.currentTimeMillis();
         }
     }
-    
+
+    public static long toMillis (int secs) {
+        return 1000*(long) secs;
+    }
+    public static long toMillis (String secs) {
```

Comment:
This is even less obvious that it is seconds to millis,  and who is calling this anyways?